lib/langchain/llm/response/anthropic_messages_response.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+module Langchain::LLM
+  class AnthropicMessagesResponse < BaseResponse
+    def model
+      raw_response.dig("model")
+    end
+
+    def chat_completions
+      raw_response.dig("content")
+    end
+
+    def chat_completion
```

Comment:
Could you please see if you could reuse the `AnthropicResponse` class I added here? https://github.com/patterns-ai-core/langchainrb/pull/580/files

Reply:
@andreibondarev Double checking tests, but it is working in my dev enviroment!